waiter/src/waiter/util/utils.clj

```diff
@@ -865,3 +873,28 @@
   [names]
   (let [filter-fns (map str->filter-fn names)]
     (fn [value] (some #(%1 value) filter-fns))))
+
+(defn string-yes?
+  "Check if string matches a yes-like value."
+  [s]
+  (some? (re-find #"^(?:[TtYy]|[Oo][Nn])" (str s))))
+
+(defn string-no?
+  "Check if string matches a no-like value."
+  [s]
+  (some? (re-find #"^(?:[FfNn]|[Oo][Ff])" (str s))))
+
+(defn raven-response-flags
```

Comment:
We should remove `raven` references from code and refer to them as `reverse-proxy`

Reply:
Resolved this by consistently using `raven` as the name for the sidecar (see #1437).